website/blog/2018-09-13-decorators.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "Article about Decorators"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2018-09-13 23:00:00
+categories: announcements
+share_text: "Article about Decorators"
+---
+
+> TODO: Better title. Help pls :pray:
```

Comment:
I :pray: am :pray: bad :pray: at :pray: choosing :pray: titles.

Reply:
"TC39 Standards-track Decorators in Babel" ?